codec/src/main/java/io/netty/handler/codec/compression/Snappy.java

```diff
@@ -16,6 +16,7 @@
 package io.netty.handler.codec.compression;
 
 import io.netty.buffer.ByteBuf;
+import io.netty.util.internal.shaded.org.jctools.util.Pow2;
```

Comment:
maybe we could also expose this via `PlatformDependent` and so remove the whole import of a shaded class here

Reply:
Tell me what you prefer, if we want to use the shaded jar then this fix provided [here](https://youtrack.jetbrains.com/issue/IDEA-93855/Please-add-support-for-maven-shade-plugin-renaming#focus=Comments-27-6055272.0-0) works

Reply:
I think I would add it to either PlatformDepend or MathUtil

Reply:
I've just realised there's already that method in MathUtill.findNextPositivePowerOfTwo 

let me use that

Reply:
https://github.com/netty/netty/blob/14e00f50d13d112a1a7a8eaa693c2363adeee629/common/src/main/java/io/netty/util/HashedWheelTimer.java#L337

at this point worth (or not?) changing this as well?

Reply:
yes... just lets use a separate PR

Reply:
https://github.com/netty/netty/pull/13833